packages/jest-jasmine2/src/jasmine/jasmine_light.js

```diff
@@ -42,7 +42,7 @@ import SpyRegistry from './spy_registry';
 import Suite from './Suite';
 import Timer from './Timer';
 
-export const create = function(createOptions: Object) {
+exports.create = function(createOptions: Object) {
```

Comment:
y?

Reply:
Tried to `// $FlowFixMe` [this line](https://github.com/facebook/jest/pull/5602/files/7dc5c0411bf96f03da17d5a7c2da2cbd88c80b1d#diff-5cb2cdb5d297e7ccdc422c99a384512cL69), but didn't work. Decided to keep it consistently bad :p

Reply:
For the record, Flow started to fail this line with:
```
Unable to determine module type (CommonJS vs ES) if both an export statement and
module.exports are used in the same module!
```